pkgs/development/lisp-modules/nix-cl.nix

```diff
@@ -222,6 +222,7 @@ let
       patches = [];
       propagatedBuildInputs = args.propagatedBuildInputs or []
           ++ lispLibs ++ javaLibs ++ nativeLibs;
+      meta = (args.meta or {}) // { maintainers = lib.teams.lisp.members; };
```

Comment:
Hmm, this doesn't allow to have extra maintainers for e.g. Nyxt, right?

Reply:
Currently nyxt doesn't have this problem, because it's made from a separate `mkDerivation` call, and uses `sbclPackages.nyxt` only as a "source". Stumpwm does have this problem, because it comes directly from `sbclPackages`, but there we could use `overrideAttrs` to change `meta.maintainers`.

Reply:
Maybe just say `maintainers = args.meta.maintainers or lib.teams.lisp.members` ? (alternatively, `or []` and append the team)

Reply:
Ok 6817b72ca3cdb8e5edd9ff84a1dc549de1c683f3